### ISSUE TITLE
Invalid default value for 'modified'

### ISSUE THREAD

**Issue by gllmhyt** [chimo/gs-nodeinfo#4]
Getting this error:
```php
PEAR Error: DB Error: unknown error (CREATE TABLE usage_stats (
type varchar(191) comment 'Type of countable entity',
count int default 0 comment 'Number of entities of this type',
modified timestamp default 'CURRENT_TIMESTAMP' comment 'date this record was modified',
PRIMARY KEY (type),
CONSTRAINT usage_stats_key UNIQUE (type)
) ENGINE=InnoDB CHARACTER SET utf8mb4 COLLATE utf8mb4_bin [nativecode=1067 ** Invalid default value for 'modified'])
```
Seems like the `''` around CURRENT_TIMESTAMP are superfluous.

**Comment by chimo** [comment]
Fixed via c9787f8c46e736

Note to self: apparently this wasn't an issue in [this repo](https://notabug.org/diogo/gnu-social) -- which may or may not be the official one now? -- because of [this commit](https://notabug.org/diogo/gnu-social/commit/9a39ebe66f11a5a0c8158d83bc37dec207b9cb8c).

**Comment by gllmhyt** [comment]
I discovered by heading to git.gnu.io that GitHost.io will be shut down on June 1, 2019 and at that point git.gnu.io will be unreachable and all data will be irrevocably deleted.

Thank you for pointing out the fork made by @diogogithub, it seems to be quite active! I will see if I can use it. The latest commit upstream has been your fix 9 months ago, that's sad to see the project quite abandonned by Mikael (@mmn) and bugs and merge requests piling up like when it was Evan's StatusNet.
